lib/react/rails/railtie.rb

```diff
@@ -46,17 +46,17 @@ class Railtie < ::Rails::Railtie
                      tmp_path.join('react.js'))
         FileUtils.cp(::React::Source.bundled_path_for('JSXTransformer.js'),
                      tmp_path.join('JSXTransformer.js'))
-        app.assets.prepend_path tmp_path
+        app.config.assets.paths.unshift(tmp_path)
```

Comment:
Could you just wrap this entire "setup_vendor" thing in `config.assets.configure` instead of using an `initializer`?

Something like

``` ruby
config.assets.configure do |env|
  tmp_path = env.root.join('tmp/react-rails')
  # ...
  env.prepend_path tmp_path
end
```